src/server/mod.rs

```diff
@@ -128,7 +128,7 @@ macro_rules! build_app {
             )
             // Tokenserver
             .service(
-                web::resource(&cfg_path("/1.0/sync/1.5")).route(web::get().to(tokenserver::get)),
+                web::resource("/1.0/sync/1.5".to_string()).route(web::get().to(tokenserver::get)),
```

Comment:
Now that this route does something other than nothing, we should probably disable this during development somehow on the live service. A setting or maybe something as simple as detecting a debug build (but note the docker-compose tests in ci run against release mode)

Reply:
The docker-compose tests don't hit this tokenserver yet, so that should be fine.

Reply:
You have to go far out of your way to configure your firefox to use this new tokenserver route and it's not documented anywhere, so does this really buy anything?

Reply:
I'll admit I'm being pedantic but I think it makes sense at some point to disable the route when it's not needed anyway (thinking about it more, presence of `TOKENSERVER_DATABASE_URL` would be a good toggle).

Definitely a long shot anyone hits this endpoint but I was more concerned about it being untested/load tested, particularly w/ this handler potentially loading the Python runtime. So I'd rather just disable it sooner rather than later so there's one less thing on prod to worry about while it's still under development.